conda_build/skeletons/pypi.py

```diff
@@ -313,17 +302,16 @@ def skeletonize(packages, output_dir=".", version=None, recursive=False,
             base_url = urlsplit(pypi_url)
             base_url = "://".join((base_url.scheme, base_url.netloc))
             ordered_recipe['source']['url'] = urljoin(base_url, ordered_recipe['source']['url'])
+            ordered_recipe['source']['sha256'] = d['hash_value']
 
             if d['entry_points']:
                 ordered_recipe['build']['entry_points'] = d['entry_points']
 
             if noarch_python:
                 ordered_recipe['build']['noarch'] = 'python'
 
-            ordered_recipe['build']['script'] = 'python setup.py install ' + ' '.join(setup_options)
-            if any(re.match(r'^setuptools(?:\s|$)', req) for req in d['build_depends']):
-                ordered_recipe['build']['script'] += ('--single-version-externally-managed '
-                                                      '--record=record.txt')
+            ordered_recipe['build']['script'] = ('python -m pip install . --no-deps '
```

Comment:
Should this be `$PYTHON` instead or is that not required?

Reply:
not strictly.  python should always be on PATH always in the env.  If you make it $PYTHON, that's more definite, but it's not strictly necessary.  I'll go ahead and change it, though.

Reply:
Hm.. although I'm usually in favor of generalizing and assuring correctness, I'd rather have just `python` instead of `"${PYTHON}"`/`"%PYTHON%"` here. See comments from https://github.com/conda-forge/conda-forge.github.io/pull/551#issuecomment-378331154 onward.
Just `python` is also the prevalent thing on `aggregate` and `feedstocks`.

Reply:
There was some reason why Intel had this pattern.  I can't remember.  It did have some advantage over depending on PATH.

The current code uses jinja2 instead of env vars, and should work.  Errr... it will, when I can get to a better internet connection that doesn't give me git errors.